packages/cli/src/databases/dsl/Column.ts

```diff
@@ -0,0 +1,117 @@
+import type { Driver, TableColumnOptions } from 'typeorm';
+
+export class Column {
+	private type: 'int' | 'boolean' | 'varchar' | 'text' | 'json' | 'timestamp' | 'uuid';
+
+	private isGenerated = false;
+
+	private isNullable = true;
+
+	private isPrimary = false;
+
+	private length: number | 'auto';
+
+	private defaultValue: unknown;
+
+	constructor(private name: string) {}
+
+	get bool() {
+		this.type = 'boolean';
+		return this;
+	}
+
+	get int() {
+		this.type = 'int';
+		return this;
+	}
+
+	varchar(length?: number) {
+		this.type = 'varchar';
+		this.length = length ?? 'auto';
+		return this;
+	}
+
+	get text() {
+		this.type = 'text';
+		return this;
+	}
+
+	get json() {
+		this.type = 'json';
+		return this;
+	}
+
+	timestamp(length?: number) {
+		this.type = 'timestamp';
+		this.length = length ?? 'auto';
+		return this;
+	}
```

Comment:
This is what we have right now:

```ts
const timestampSyntax = {
	sqlite: "STRFTIME('%Y-%m-%d %H:%M:%f', 'NOW')",
	postgresdb: 'CURRENT_TIMESTAMP(3)',
	mysqldb: 'CURRENT_TIMESTAMP(3)',
	mariadb: 'CURRENT_TIMESTAMP(3)',
}[dbType];

export const datetimeColumnType = dbType === 'postgresdb' ? 'timestamptz' : 'datetime';

const tsColumnOptions = {
	precision: 3,
	default: () => timestampSyntax,
};
```

Are `CURRENT_TIMESTAMP` for MySQL `STRFTIME('%Y-%m-%d %H:%M:%f', 'NOW')` for sqlite, `timestamptz` for Postgres accounted for?